libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -142,4 +142,40 @@ describe('SegmentedControlComponent', () => {
       expect(component.value).toBe(items[2]);
     });
   });
+
+  describe("in 'compact chip' mode", () => {
+    beforeEach(() => {
+      spectator.setInput('mode', SegmentedControlMode.compactChip);
+      spectator.detectChanges();
+    });
+
+    it("should have a 'compact chip' mode when created", () => {
+      expect(component.mode).toBe(SegmentedControlMode.compactChip);
+    });
+
+    it('should not have an ion-segment control', () => {
+      expect(spectator.queryHost('ion-segment')).toBeNull();
+    });
+
+    it('should not have any segments buttons', () => {
+      expect(spectator.queryHostAll('ion-segment-button').length).toBe(0);
+    });
+
+    it('should have a segment chip per item', () => {
+      expect(spectator.queryHostAll('kirby-chip').length).toBe(items.length);
+    });
+
+    it('should call onSegmentSelect when clicking a different segment chip', () => {
+      expect(component.value).toBe(items[1]);
+      spyOn(component, 'onSegmentSelect');
+      spectator.dispatchMouseEvent('kirby-chip:first-of-type', 'click');
+      expect(component.onSegmentSelect).toHaveBeenCalled();
+    });
+
+    it('should set value when clicking a different segment chip', () => {
+      expect(component.value).toBe(items[1]);
+      spectator.dispatchMouseEvent('kirby-chip:last-of-type', 'click');
```

Comment:
Consider readability
```suggestion
      spectator.dispatchMouseEvent('kirby-chip:last-of-type', 'click');
```